### ISSUE TITLE
ignore query string when using spa.asset-test-fs

### ISSUE THREAD

**Issue by 5im-0n** [lwsjs/spa#2]
Hi @75lb, is there a way to check only for the filename part when using `spa.asset-test-fs`?
For example say I include a file with
```
<script type="javascript" src="http://127.0.0.1:8000/myscript.js?key=123"></script>
```
so `spa.asset-test-fs` should look for `myscript.js`, and not for `myscript.js?key=123`.

Maybe something like
```
        } else if (options.spaServeExisting && route !== '/') {
          const fs = require('fs')
          const path = require('path')
          const filePath = path.join(root, require('url').parse(route).pathname) //<------
          isStatic = fs.existsSync(filePath)
        }
```
(i did not test this). What do you think?

**Comment by 75lb** [comment]
yes,  I'll fix this today thanks

**Comment by 75lb** [comment]
Fixed in lws-spa v1.0.1 👍

**Comment by 5im-0n** [comment]
Awesome. Thanks!
